src/app/Home.module.css

```diff
@@ -1,33 +1,14 @@
-.landingBackground h1 {
-    font-size: 2em;
-    margin-bottom: 1rem;
-    margin-left: 1em;
-    margin-right: 1em;
-}
-  
-.landingBackground h2 {
-    font-size: 1.5em;
-    margin-bottom: 0.5rem;
-    margin-left: 1em;
-    margin-right: 1em;
-}
-
-.landingBackground p {
-    margin-bottom: 0.5rem;
-    margin-left: 1em;
-    margin-right: 1em;
-}
-  
-.centerText {
+.content {
+    margin: 0 auto;
+    max-width: 800px;
     text-align: center;
 }
-  
+
 .landingBackground {
+    padding: 1px; /* hack to prevent margin collapse */

```

Comment:
This is... dumb. There are better way to do this, but the background image doesn't really work with the navbar (because the navbar takes up height causing the image to go past the bottom of the page) and it's a larger overhaul to address that. Will revisit this problem later; we may end up changing how this background image is used anyways.